### PR TITLE
chore: add failed checklist

### PR DESCRIPTION
**Pull request by septs**
## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: docs/failed-checklist.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Failed Checklist
```
Comment (Jack-Works) [inline docs/failed-checklist.md]: Build failure / Runtime crash checklist

File: docs/failed-checklist.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Build failed
+
+1. Run `gulp clean`, Cleanup Caching
```
Comment (Jack-Works) [inline docs/failed-checklist.md]:
```suggestion
1. Run `npx gulp clean`, Cleanup cache and built files
```

File: docs/failed-checklist.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Install failed
```
Comment (Jack-Works) [inline docs/failed-checklist.md]: CI Error: Cannot connect to git@github.com, public key rejected

File: docs/failed-checklist.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+1. Run `gulp fix-lockfile`, Rebuild lockfile
```
Comment (Jack-Works) [inline docs/failed-checklist.md]:
```suggestion
1. Run `npx gulp fix-lockfile`
```

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: docs/failed-checklist.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Build failure / Runtime Crash Checklist
+
+## Build failed
```
Comment (Jack-Works) [inline docs/failed-checklist.md]: actually this command also works for runtime crash

File: docs/failed-checklist.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Build failure / Runtime Crash Checklist
+
+## Runtime crash
```
Comment (Jack-Works) [inline docs/failed-checklist.md]:
```suggestion
# Checklist

## Build error / Runtime crash
```

**Review by Jack-Works — DISMISSED** [review]

File: docs/failed-checklist.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# Checklist
+
+## Build error / Runtime crash
+
+1. Run `npx gulp clean`, Cleanup cache and built files
```
Comment (guanbinrui) [inline docs/failed-checklist.md]: build files.

**Review by guanbinrui — DISMISSED** [review]

File: docs/failed-checklist.md (hunk 7 above)
Comment (septs) [reply docs/failed-checklist.md]:
```suggestion
1. Run `npx gulp clean`, Cleanup cache and build files
```

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**